### PR TITLE
chore: upgrade eslint

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
+): Vector3[] => {
   const geometryVertices: Vector3[] = [];
   for (let i = 0; i < 3; i++) {
     // @ts-expect-error - This expression is not constructable. Type 'Function' has no construct signatures.
     const typedArray = new attribute.value.constructor(3);
-    // console.log('typedArray----', typedArray);
     const subarray = attribute.value.subarray(
-      (offset + i) * attribute.size,
-      (offset + i) * attribute.size + attribute.size
+      offset + i * attribute.size,
```
Comment (maxkuznetsov-actionengine) [inline src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts:22]: Formula is changed, is it correct?

**Review by maxkuznetsov-actionengine — APPROVED** [review]

File: src/components/arcgis-wrapper/arcgis-wrapper.tsx [hunk 2]
```diff
@@ -288,7 +288,7 @@ export const ArcgisWrapper = ({
       delete tile.content.featureIds;
     }
     if (showDebugTextureRef.current) {
-      selectDebugTextureForTile(tile, uvDebugTextureRef.current);
+      void selectDebugTextureForTile(tile, uvDebugTextureRef.current);
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/arcgis-wrapper/arcgis-wrapper.tsx:291]: Yes. This is a eslint requirement for hanging promises https://typescript-eslint.io/rules/no-floating-promises/

File: src/components/comparison/comparison-side/comparison-side.tsx [hunk 3]
```diff
@@ -243,15 +243,15 @@ export const ComparisonSide = ({
       });
     }
 
-    dispatch(
+    void dispatch(
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/comparison/comparison-side/comparison-side.tsx:246]: Yes. This is a eslint requirement for hanging promises https://typescript-eslint.io/rules/no-floating-promises/

File: src/components/header/header.spec.tsx [hunk 4]
```diff
@@ -2,13 +2,14 @@ import { renderWithTheme } from "../../utils/testing-utils/render-with-theme";
 import { Header } from "./header";
 import { screen } from "@testing-library/react";
 import { useAppLayout } from "../../utils/hooks/layout";
+import type { ReactChildren } from "react";
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/header/header.spec.tsx:5]: There were over 1000 errors. The most of fixes is `eslint \"src/**\" --fix`. I can't fix them all manually.

File: src/components/header/header.tsx [hunk 5]
```diff
@@ -114,7 +116,7 @@ export const Header = ({
           showHelp={showHelp}
           onHelpClick={onHelpClick}
         />
-      )}
+          )}
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/header/header.tsx:119]: `eslint \"src/**\" --fix`. We don't have prettier in the project. We'd probably use some eslint formatter in future.

File: src/components/layers-panel/arcgis-control-panel.tsx [hunk 6]
```diff
@@ -70,10 +70,10 @@ export const ArcGisControlPanel = ({
 
   const onArcGisActionClick = () => {
     if (isLoggedIn) {
-      dispatch(getArcGisContent());
+      void dispatch(getArcGisContent());
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/layers-panel/arcgis-control-panel.tsx:73]: It is a https://typescript-eslint.io/rules/no-floating-promises/ requirement

File: src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx [hunk 7]
```diff
@@ -273,7 +275,7 @@ export const ArcGisImportPanel = ({ onImport, onCancel }: InsertLayerProps) => {
                 }}
               />
             </Radio>
-          )}
+              )}
```
[the remark this answers is not in the document]
Comment (belom88) [reply src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx:278]: eslint --fix

File: src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts (hunk 1 above)
Comment (belom88) [reply src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts:22]: Yes. I've found a mistake when I fixed a test for this function.

Comment (belom88) [reply src/utils/debug/validation-utils/attributes-validation/triangles-calculation.ts:22]: Yes, I've found a mistake when I fixed the retun type and the test

**Pull request merged**